modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java

```diff
@@ -1558,16 +1559,53 @@ private void _validateListTypeEntriesValues(
 		}
 	}
 
+	private void _validateObjectFieldIntegerTypeSize(
+			Map.Entry<String, Serializable> entry)
+		throws ObjectEntryValuesException {
+
+		Serializable entryValue = entry.getValue();
+
+		if (!StringUtil.equals(entryValue.toString(), "")) {
```

Comment:
What do you think about creating a variable for "entryValue.toString()" and use the method "isEmpty()"?